src/azure-cli/azure/cli/command_modules/network/private_link_resource_and_endpoint_connections/custom.py

```diff
@@ -76,18 +84,21 @@ def approve_private_endpoint_connection(cmd, resource_group_name, resource_name,
 
 def reject_private_endpoint_connection(cmd, resource_group_name, resource_name, resource_provider,
                                        name, rejection_description=None):
+    _check_manually_approve_support(TYPE_CLIENT_MAPPING, resource_provider)
     client = _get_client(TYPE_CLIENT_MAPPING, resource_provider)
     return client.reject_private_endpoint_connection(cmd, resource_group_name,
                                                      resource_name, name,
                                                      rejection_description)
 
 
 def remove_private_endpoint_connection(cmd, resource_group_name, resource_name, resource_provider, name):
+    _check_manually_approve_support(TYPE_CLIENT_MAPPING, resource_provider)
     client = _get_client(TYPE_CLIENT_MAPPING, resource_provider)
     return client.remove_private_endpoint_connection(cmd, resource_group_name, resource_name, name)
 
 
 def show_private_endpoint_connection(cmd, resource_group_name, resource_name, resource_provider, name):
+    _check_manually_approve_support(TYPE_CLIENT_MAPPING, resource_provider)
```

Comment:
Cannot user show this private endpoint connection?

Reply:
Yes, service does not provide the api yet. `Show` command should use the same api with `approve`, `reject`, and `delete`. `List` command could work because it falls back to use the resource api [here](https://github.com/Azure/azure-cli/blob/dev/src/azure-cli/azure/cli/command_modules/network/private_link_resource_and_endpoint_connections/resource_providers/__init__.py#L134).